package.json

```diff
@@ -912,7 +912,7 @@
         "ts-node": "^8.5.2",
         "vsce": "^1.69.0",
         "vscode-nls-dev": "^3.3.1",
-        "vscode-test": "^1.3.0",
+        "vscode-test": "^1.4.0",
```

Comment:
Does this give any benefits directly related to this PR or is it just a nice-to-have? Just curious since we haven't touched this in a while.

Reply:
This could be in a separate PR if needed. It is being bumped because we suspect that when our tests fail with sigterm issues, this version will console log additional details.